src/main/scala/com/gu/subscriptions/cas/config/Configuration.scala

```diff
@@ -47,7 +48,11 @@ object Configuration {
 
   def productIds(env: String): com.gu.memsub.subsv2.reads.ChargeListReads.ProductIds =
     SubsV2ProductIds(touchpointConfig.getConfig("zuora.productIds"))
-
+  object EmergencyTokens {
+    private val conf = appConfig.getConfig("cas.emergency.subscriber.auth")
+    val prefix = conf.getString("prefix")
+    val secret = conf.getString("secret")
+  }
```

Comment:
These two entries need to be added to the configuration file before this PR is merged